index.html

```diff
@@ -1,15 +1,13 @@
 <!doctype html>
-
 <!--
-    @license
-    Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
-    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-    Code distributed by Google as part of the polymer project is also
-    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
```

Comment:
It used to indent but it doesn't since this commit (https://github.com/PolymerLabs/seed-element/commit/fdd12dd3fe7ccbd1cc8fbce93bc7dd514b9c5954), but I just realized it yesterday.